pennylane_sf/simulator.py

```diff
@@ -122,6 +123,20 @@ def expval(self, expectation, wires, par):
 
         return ex
 
+    def var(self, observable, wires, par):
+        """Evaluate the variance of an observable.
+
+        Args:
+            observable (str): name of the observable
+            wires (Sequence[int]): subsystems the observable is evaluated on
+            par (tuple): parameters for the observable
+
+        Returns:
+            float: variance value
+        """
+        _, var = self._observable_map[observable](self.state, wires, par)
```

Comment:
just curious, is it more efficient to set up separate functions for expectation values and variances, rather than a single function which returns both, even if only one is needed?

Reply:
This is an SF limitation --- all state methods return a tuple of expectations and variance, so it doesn't sync well with Pennylane's one-or-the-other approach. Worth making an issue for this and seeing if there is a better solution.

Reply:
k, so the efficiency gain would be found within SF itself, not plugin. Got it.